README.md

```diff
@@ -31,103 +31,106 @@ export PATH=$PATH:/myworkspace/dkan-tools/bin
 
 Once you are working in a valid project folder (see next section) you can type `dktl` at any time to see a list of available commands.
 
-## Starting a DKAN 1.x project
-To start a project with `dktl`, create a project directory.
+## Local development
+The React front end uses the DKAN API to build pages, so we will want to mimic a production environment. The environment variables are [defined in `.env` files](https://github.com/GetDKAN/data-catalog-frontend/blob/master/.env.production#L1), the default value is _"dkan"_, you can adjust these files as necessary.
+
+Setup and start the proxy:
+
+- Add `dkan` to `/etc/hosts`
+- Start the proxy: `docker run -d -p 80:80 -v /var/run/docker.sock:/tmp/docker.sock:ro jwilder/nginx-proxy`
+
+## Starting a new project
+1. To start a project with `dktl`, create a project directory.
 
 ```bash
 mkdir my_project && cd my_project
 ```
 
-Inside the project directory, initialize your project.
+2. Inside the project directory, initialize your project.
 
 ```bash
 dktl init
 ```
 
 This will automatically start up the Docker containers, which can also be started manually with `dktl docker:compose up -d`. Any other docker-compose commands can be run via `dktl docker:compose <args>` or simply `dktl dc <args>`.
 
-After initialization, we want to get DKAN ready. We can use `git clone` (recommended if you are working directly on DKAN core and will want to commit and push changes to the DKAN project) or download a tarball of the DKAN source from [GitHub](https://github.com/GetDKAN/dkan), but the easiest method is using this command:
+3. Get Drupal, only versions 8.8 or above are supported:
 
 ```bash
-dktl dkan:get <version_number>
+dktl get <drupal-version>
 ```
 
-Versions of DKAN look like this: ``7.x-1.15.3``. We can see all of [DKAN's releases](https://github.com/getDkan/dkan/releases) in Github.
-
-Now run the "make" command:
+4. Get Drupal dependencies and DKAN modules. This will create the symlinks necesarry to create a working Drupal site under _/docroot_.
 
 ```bash
 dktl make
 ```
 
-The `make` command will get all of DKAN's dependencies _including_ Drupal core. It will also create all the symlinks necesarry to create a working Drupal site under _/docroot_.
+  - Flag options:
+      * If you are working directly on the DKAN project or one of its libraries and want to be able to commit changes and submit pull requests, use the `--prefer-source` option with `dktl make`. This option will be passed directly to Composer; see the [Composer CLI documentation](https://getcomposer.org/doc/03-cli.md#command-line-interface-commands) for more details.
+      * To install the React frontend application add the `--frontend` option with `dktl make`. This will download the app to `src/frontend` and symlink the files to `docroot/data-catalog-frontend`.
```

Comment:
Should we specify how to build the frontend? Maybe at least a link to that? Maybe not at all as it is not dkan-tools specific?